<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForumPage</class>
 <widget class="QWidget" name="ForumPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>187</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="miscGBox">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QGridLayout" name="miscGBoxGLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="setMsgToReadOnActivate">
        <property name="text">
         <string>Set message to read on activate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="expandNewMessages">
        <property name="text">
         <string>Expand new messages</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="loadEmbeddedImages">
        <property name="text">
         <string>Load embedded images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="mainVSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="tabsGBox">
     <property name="title">
      <string>Tabs</string>
     </property>
     <layout class="QVBoxLayout" name="tabsGBoxVLayout">
      <item>
       <widget class="GroupFrameSettingsWidget" name="groupFrameSettingsWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupFrameSettingsWidget</class>
   <extends>QWidget</extends>
   <header>gui/settings/GroupFrameSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
